packages/sanity/src/core/releases/hooks/useVersionOperations.tsx

```diff
@@ -71,8 +72,34 @@ export function useVersionOperations(): VersionOperationsValue {
       })
     }
   }
+
+  const handleUnpublishVersion = async (documentId: string) => {
+    try {
+      await unpublishVersion(documentId)
+
+      toast.push({
+        closable: true,
+        status: 'success',
+        description: (
+          <Translate
+            t={t}
+            i18nKey={'release.action.unpublish-version.success'}
+            values={{title: document.title as string}}
```

Comment:
NIT: it doesn't look like we need a `title` in this translation string

Reply:
It probably should, I'll update!